transport/tchannel/options.go

```diff
@@ -50,17 +50,18 @@ var _ Option = (TransportOption)(nil)
 // peer lists.
 // TODO update above when NewTransport is real.
 type transportOptions struct {
-	ch                    Channel
-	tracer                opentracing.Tracer
-	logger                *zap.Logger
-	addr                  string
-	listener              net.Listener
-	dialer                func(ctx context.Context, network, hostPort string) (net.Conn, error)
-	name                  string
-	connTimeout           time.Duration
-	connBackoffStrategy   backoffapi.Strategy
-	originalHeaders       bool
-	nativeTChannelMethods NativeTChannelMethods
+	ch                      Channel
+	tracer                  opentracing.Tracer
+	logger                  *zap.Logger
+	addr                    string
+	listener                net.Listener
+	dialer                  func(ctx context.Context, network, hostPort string) (net.Conn, error)
+	name                    string
+	connTimeout             time.Duration
+	connBackoffStrategy     backoffapi.Strategy
+	originalHeaders         bool
+	nativeTChannelMethods   NativeTChannelMethods
+	disableRpcServiceHeader bool
```

Comment:
It is not clear if it disables the serviceHeader in the response. Can we make it more explicit like excludeServiceHeaderInResponse or something related?

Reply:
Using excludeServiceHeaderInResponse as suggested. thanks